app/views/templates/run_command.erb

```diff
@@ -0,0 +1,13 @@
+<%#
+kind: job_template
+name: Run Command - SSH Default
+job_name: RunCommand
+provider_type: Ssh
+template_inputs:
+- name: command
+  description: Command to run on the host
+  input_type: user
+  required: true
+%>
+
```

Comment:
Every seeded template has two blank lines at the beginning:

```


<%= input("command") %>
```

would be nice to just start with the code, but maybe this is something that would need to be handled in Foreman core?